packages/feedback/src/util/sendFeedbackRequest.ts

```diff
@@ -34,6 +35,10 @@ export async function sendFeedbackRequest({
     type: 'feedback',
   };
 
+  if (source === FEEDBACK_WIDGET_SOURCE) {
+    scope.setLevel('info');
+  }
```

Comment:
This is not good because we mutate the scope, so that will also affect other events that have the same scope (unless I am missing something and we are always forking a scope for this scenario?). So we should do one of these:

1. Fork a scope here, and mutate that one:
```js
withScope(scope => 
  scope.setLevel(...);
  prepareFeedbackEvent({ scope, ... })
})
```
2. put the level directly on the feedback event somehow

Reply:
Good catch! I updated to use `pushScope` and `popScope` -- I don't love the `withScope` API.... I wish `withScope` returned the new scope.